client/template.js

```diff
@@ -1,7 +1,7 @@
-module.exports = async(name, title = '', props = {})=>{
+module.exports = async(name, title = '', props = {}, lang='en')=>{
 	return `
 <!DOCTYPE html>
-<html>
+<html lang="${lang}">
```

Comment:
I don't think this is where we want to apply the `lang` attribute. Since users will be setting this from the `metadata` panel, it will cause the brew document to update its HTML, but not trickle up to the full web page here without a page refresh. This probably needs to be applied at the `brewRenderer.jsx` page, possibly in the `initialContent` variable there...